packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -0,0 +1,30 @@
+import moment from 'moment';
+
+class DateUtil {
+  static createSafeMoment(date, format) {
```

Comment:
This function feels misnamed. When I think createSafeMoment, it makes me think that it's returning a moment object, whereas if the moment object isn't valid, it returns the value that was passed in. Perhaps createSafeDate would be a better option than createSafeMoment?